### PR TITLE
Fix signing for 2.2 site extensions

### PR DESCRIPTION
**Pull request by pakrym**
2.2 build is more similar to 3.1 then 2.1 so switch it to use the same patterns as we do in 3.0.

### CODE REVIEW

File: src/SiteExtensions/LoggingBranch/LoggingBranch.csproj [hunk 1]
```diff
@@ -23,11 +24,23 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <HostingStartupRuntimeStoreTargets Include="netcoreapp2.2" Runtime="win7-x64" />
-    <HostingStartupRuntimeStoreTargets Include="netcoreapp2.2" Runtime="win7-x86" />
+    <HostingStartupRuntimeStoreTargets Include="netcoreapp3.0" Runtime="$(SharedFxRid)" />
```
Comment (natemcmaster) [inline src/SiteExtensions/LoggingBranch/LoggingBranch.csproj]:
```suggestion
    <HostingStartupRuntimeStoreTargets Include="netcoreapp2.2" Runtime="$(SharedFxRid)" />
```

File: src/SiteExtensions/LoggingBranch/LoggingBranch.csproj [hunk 2]
```diff
@@ -23,11 +24,23 @@
...
+
     <HostingStartupPackageReference Include="Microsoft.AspNetCore.AzureAppServices.HostingStartup" Version="$(PackageVersion)" />
 
     <PackageReference Include="Internal.AspNetCore.Sdk" PrivateAssets="All" Version="$(InternalAspNetCoreSdkPackageVersion)" />
+
+    <ProjectReference Include="..\..\Framework\Microsoft.AspNetCore.App\pkg\Microsoft.AspNetCore.App.pkgproj">
```
Comment (natemcmaster) [inline src/SiteExtensions/LoggingBranch/LoggingBranch.csproj]: Do you also need to reference Microsoft.AspNetCore.All?

Comment (pakrym) [reply src/SiteExtensions/LoggingBranch/LoggingBranch.csproj]: It trims against `.App` so this should be enough.

**Review by natemcmaster — APPROVED** [review]

**Review by muratg — APPROVED** [review]

**Comment by pakrym** [comment]
Verified on aspnetci

**Pull request merged**